src/types/ResponseError.ts

```diff
@@ -0,0 +1,4 @@
+export interface ResponseError {
+  Response: 'False',
+  Error: string,
+}
```

Comment:
looks like we shouldn't set type, to server's meta data

Reply:
This file was already in the folder